vertx-pojo-mapper-common/src/main/java/de/braintags/vertx/jomnigate/dataaccess/query/impl/IndexedField.java

```diff
@@ -11,34 +11,43 @@
 public class IndexedField implements IIndexedField {
 
   private String fieldName;
-  private String columnName;
 
   public IndexedField(String name) {
-    this(name, name);
+    this.fieldName = name;
   }
 
-  public IndexedField(String fieldName, String columnName) {
-    this.fieldName = fieldName;
-    this.columnName = columnName;
+  @Override
+  public String getFieldName() {
+    return fieldName;
   }
 
   @Override
-  public String getFieldName() {
+  public String toString() {
     return fieldName;
   }
 
-  /*
-   * (non-Javadoc)
-   * 
-   * @see de.braintags.vertx.jomnigate.dataaccess.query.IIndexedField#getColumnName()
-   */
   @Override
-  public String getColumnName() {
-    return columnName;
+  public int hashCode() {
+    final int prime = 31;
+    int result = 1;
+    result = prime * result + ((fieldName == null) ? 0 : fieldName.hashCode());
+    return result;
   }
 
   @Override
-  public String toString() {
-    return fieldName;
+  public boolean equals(Object obj) {
+    if (this == obj)
+      return true;
+    if (obj == null)
```

Comment:
could it make sense to get one line like obj == null ||getClass() != obj.getClass ?